releasenotes/notes/deprecate-classical-function-3bf44ef26d984366.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations_circuits:
```

Comment:
It would be nice to mention why they are deprecated and also mention in the reno that the `BooleanExpression` can be used as replacement 🙂